tasks/steps/cleanup.yml

```diff
@@ -0,0 +1,10 @@
+---
+
+- name: 'Remove temporary files'
+  file:
+    path: '{{ filepath }}'
+    state: absent
+  loop_control:
+    loop_var: filepath
+  with_items: '{{ temporary_files }}'
+  when: cartridge_remove_temporary_files
```

Comment:
Maybe it would be better to run this task for one instance per machine?

Would it work for smth like "{{ temporary_files + ['<some-instance-specific-name>'] }}"?

Reply:
> Maybe it would be better to run this task for one instance per machine?
Yes.

> Would it work for smth like "{{ temporary_files + [''] }}"?
For what? Seems that empty string is an incorrect path.

Reply:
I've mistaken in comment, sorry.

I mean that we can run this task for one instance per machine, BUT in this case `"{{ temporary_files + ['<file special for instance>'] }}"` wouldn't work, or no?

For example, user can say `"{{ temporary_files + [ '{{ inventory_hostname }}.tmp '] }}"`.

Reply:
It's possible use different lists for each instance, so it's impossible to run this task for each machine instead instance